### PR TITLE
AWS marketplace requirements - Critical update!

### PR DESCRIPTION
**Pull request by Colin1841**
Jira: https://jira.d2iq.com/browse/D2IQ-85351

There are 3 critical updates here, required by AWS:

1. Add architecture diagrams
2. Add note & link to https://calculator.aws/#/
3. Add note & link to https://aws.amazon.com/premiumsupport/knowledge-center/manage-service-limits/

There are more diagrams available here, if another one of these makes more sense we can change them out:
DKP Enterprise diagrams: https://lucid.app/lucidchart/eee59bf2-663f-4c94-bddb-a428cef82aa4/edit?invitationId=inv_cd3f70c6-6d61-4a4a-8b6c-6b947ce19c88&page=ZWfU7inD.C2N#

DKP Essential diagrams: https://lucid.app/lucidchart/92c54bc0-09a1-4576-883d-4ae40a4b9679/edit?invitationId=inv_66c0f799-78ac-4e14-a86e-50adb6e5f405&page=o75nu5mLZd~-#

### CODE REVIEW

**Review by vacas5 — COMMENTED** [review]
do we need to make the change in 2.1 docs as well so it shows up on the live site?

**Comment by Colin1841** [comment]
> do we need to make the change in 2.1 docs as well so it shows up on the live site?

Yes, I'll add to 2.1 once the edits are done/approved, so then we don't need to double-up any changes.

**Review by cneth — APPROVED** [review]

**Comment by msdolbey** [comment]
Some issues with the images and links - pushing a change @Colin1841

**Comment by msdolbey** [comment]
K - after that - looks good to me!

**Review by msdolbey — APPROVED** [review]
👍 

Critical Update!

**Comment by msdolbey** [comment]
FYI @Colin1841  - the reason the image links had to get updated: 
When you're calling on a file in markdown on our setup, it has to be sorta hardcoded (I saw sorta, because you can use the structure: 
`/dkp/konvoy/2.2/img/DKP_essentials.png`
The relative link of 
`../../img/DKP_essentials.png` 
just doesn't work. 🤷

**Comment by Colin1841** [comment]
@msdolbey - thanks for cleaning up the graphics links, I wasn't sure why the ../ folder structure wasn't working. I see the format now, so will use going forward as necessary.

File: pages/dkp/konvoy/2.2/choose-infrastructure/aws/index.md [hunk 1]
```diff
@@ -8,4 +8,24 @@ beta: false
...
+
+![DKP Essentials](/dkp/konvoy/2.2/img/DKP_essentials.png)
+
+This diagram shows a higher-level view of DKP Enterprise, and assumes a multi-cluster environment, where each cluster might look like the single cluster example above:
+
+![DKP Enterprise](/dkp/konvoy/2.2/img/DKP_enterprise.png)
+
+## AWS Pricing considerations
+
+Deploying AWS services can incur you cost, depending on how and what you deploy. For more information, see the [AWS Pricing Calculator](https://calculator.aws/#/).
```
Comment (Colin1841) [inline pages/dkp/konvoy/2.2/choose-infrastructure/aws/index.md:23]:
```suggestion
Deploying AWS services can incur costs to your organization, depending on how and what you deploy. For more information, see the [AWS Pricing Calculator](https://calculator.aws/#/).
```

File: pages/dkp/konvoy/2.2/choose-infrastructure/aws/index.md [hunk 2]
```diff
@@ -8,4 +8,24 @@ beta: false
...
+
+## AWS Service limits
+
+When using DKP on AWS, you need to be aware of the possibility of errors due to AWS service limits. For more information, see  the [AWS Service Limits](https://aws.amazon.com/premiumsupport/knowledge-center/manage-service-limits/).
```
Comment (Colin1841) [inline pages/dkp/konvoy/2.2/choose-infrastructure/aws/index.md:27]:
```suggestion
When using DKP on AWS, you need to be aware of the possibility of errors due to AWS service limits. For more information, see the [AWS Service Limits](https://aws.amazon.com/premiumsupport/knowledge-center/manage-service-limits/).
```

File: pages/dkp/konvoy/2.2/choose-infrastructure/aws/index.md [hunk 3]
```diff
@@ -8,4 +8,24 @@ beta: false
...
+
+This diagram shows a higher-level view of DKP Enterprise, and assumes a multi-cluster environment, where each cluster might look like the single cluster example above:
+
+![DKP Enterprise](/dkp/konvoy/2.2/img/DKP_enterprise.png)
+
+## AWS Pricing considerations
+
+Deploying AWS services can incur you cost, depending on how and what you deploy. For more information, see the [AWS Pricing Calculator](https://calculator.aws/#/).
+
+## AWS Service limits
```
Comment (Colin1841) [inline pages/dkp/konvoy/2.2/choose-infrastructure/aws/index.md:25]:
```suggestion
## AWS service limits
```

File: pages/dkp/konvoy/2.2/choose-infrastructure/aws/index.md [hunk 4]
```diff
@@ -8,4 +8,24 @@ beta: false
...
+
+This diagram shows the granular detail of a single Kubernetes cluster running in AWS Cloud:
+
+![DKP Essentials](/dkp/konvoy/2.2/img/DKP_essentials.png)
+
+This diagram shows a higher-level view of DKP Enterprise, and assumes a multi-cluster environment, where each cluster might look like the single cluster example above:
+
+![DKP Enterprise](/dkp/konvoy/2.2/img/DKP_enterprise.png)
+
+## AWS Pricing considerations
```
Comment (Colin1841) [inline pages/dkp/konvoy/2.2/choose-infrastructure/aws/index.md:21]:
```suggestion
## AWS pricing considerations
```

File: pages/dkp/konvoy/2.2/choose-infrastructure/aws/index.md [hunk 5]
```diff
@@ -8,4 +8,24 @@ beta: false
 enterprise: false
 ---
 
+The following diagrams show the two different ways you can implement Konvoy on AWS, DKP Essentials and DKP Enterprise.
```
Comment (gvasquezvargas) [inline pages/dkp/konvoy/2.2/choose-infrastructure/aws/index.md:11]: Not sure if Essentials or Essential.

Comment (Colin1841) [reply pages/dkp/konvoy/2.2/choose-infrastructure/aws/index.md:11]: That makes two of us, I've seen it both ways! I thought I was valid with "s", but maybe not!

Comment (Colin1841) [inline pages/dkp/konvoy/2.2/choose-infrastructure/aws/index.md:11]:
```suggestion
The following diagrams show the two different ways you can implement Konvoy on AWS, DKP Essential and DKP Enterprise.
```

File: pages/dkp/konvoy/2.2/choose-infrastructure/aws/index.md [hunk 6]
```diff
@@ -8,4 +8,24 @@ beta: false
...
+
+This diagram shows the granular detail of a single Kubernetes cluster running in AWS Cloud:
+
+![DKP Essentials](/dkp/konvoy/2.2/img/DKP_essentials.png)
```
Comment (Colin1841) [inline pages/dkp/konvoy/2.2/choose-infrastructure/aws/index.md:15]:
```suggestion
![DKP Essential](/dkp/konvoy/2.2/img/DKP_essentials.png)
```

**Pull request merged**